types/models.d.ts

```diff
@@ -640,11 +647,6 @@ declare module 'mongoose' {
       replacement: TRawDocType | AnyObject,
       options: QueryOptions<TRawDocType> & { includeResultMetadata: true }
     ): QueryWithHelpers<ModifyResult<ResultDoc>, ResultDoc, TQueryHelpers, TRawDocType, 'findOneAndReplace'>;
-    findOneAndReplace<ResultDoc = THydratedDocumentType>(
```

Comment:
Why did you remove `findOneAndReplace` here?

Reply:
There was a duplicate of findOneAndReplace with option ` { includeResultMetadata: true }` .
It was from old commit where someone replaced `rawResult` option to  `includeResultMetadata` which created duplicates.

I found 2 occurrences of this code. So removed one. Also removed duplicate for `findByIdAndUpdate`. 

```
findOneAndReplace<ResultDoc = THydratedDocumentType>(
      filter: FilterQuery<TRawDocType>,
      replacement: TRawDocType | AnyObject,
      options: QueryOptions<TRawDocType> & { includeResultMetadata: true }
```